includes/services/extended/instagram-basic-display.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Instagram service definition for Keyring.
+ * http://instagram.com/developer/
+ */
+
+class Keyring_Service_Instagram_Basic_Display extends Keyring_Service_OAuth2 {
+	const NAME  = 'instagram_basic_display';
```

Comment:
`NAME`s use dashes vs underscores everywhere else, let's keep that consistent.

Reply:
fixed